.circleci/generate_source.py

```diff
@@ -0,0 +1,29 @@
+from google.cloud import bigquery
+import os
+
+client = bigquery.Client.from_service_account_json(
+    f"{os.getenv('HOME')}/gcloud-service-key.json"
+)
+
+schema_name = "shopify_source_integration_tests"
+
+tables = [
+    {"seed": "shopify_customer_data", "source": "customer"},
+    {"seed": "shopify_order_line_refund_data", "source": "order_line_refund"},
+    {"seed": "shopify_order_line_data", "source": "order_line"},
+    {"seed": "shopify_order_data", "source": "order"},
+    {"seed": "shopify_product_data", "source": "product"},
+    {"seed": "shopify_order_adjustment_data", "source": "order_adjustment"},
+    {"seed": "shopify_product_variant_data", "source": "product_variant"},
+    {"seed": "shopify_refund_data", "source": "refund"},
+    {"seed": "shopify_transaction_data", "source": "transaction"},
```

Comment:
Are these the only parts of this file we will need to modify for each package?

I know there are some packages where there is a different seed file for Snowflake vs the other seeds (eg. [greenhouse scheduled_interview seed data](https://github.com/fivetran/dbt_greenhouse_source/blob/main/integration_tests/dbt_project.yml#L31). Is there a way we can call a jinja variable or even the `target.type` from the package within this python file?

Reply:
Chatted about this one, and yes this will need to be modified for each package.